cypress/e2e/tests/pages/extensions-compatibility-tests/elemental.spec.ts

```diff
@@ -0,0 +1,214 @@
+import ExtensionsPagePo from '@/cypress/e2e/po/pages/extensions.po';
+import ElementalPo from '@/cypress/e2e/po/pages/extensions-compatibility-tests/elemental.po';
+import { NamespaceFilterPo } from '@/cypress/e2e/po/components/namespace-filter.po';
+import * as jsyaml from 'js-yaml';
+
+const EXTENSION_NAME = 'elemental';
+const EXTENSION_VERSION = '2.0.0-rc2';
```

Comment:
we'll need to manually bump this every time. shouldn't we stick to latest?

Reply:
How do we define latest since it's a fixed version number?

Reply:
we would pick the first / top entry in the list (assuming versions are sorted)